Aqua Wallet

Delgado74 opened the pull request:
Se realiza tutorial de Aqua Wallet.

Delgado74 committed:
Create tutorial_aqua.md

Delgado74 committed:
Create aqua

Delgado74 committed:
Delete src/assets/images/aqua

Delgado74 committed:
Delete src/tutorial_aqua.md

Delgado74 committed:
Create inicio.jpg

Delgado74 committed:
Delete src/assets/images/aqua/inicio.jpg

Delgado74 committed:
Create inicio

Delgado74 committed:
Add files via upload

Delgado74 committed:
Create tutorial_aqua

Delgado74 committed:
Create aqua.md

Delgado74 committed:
Delete src/tutorial_aqua

Delgado74 committed:
Update aqua.md

se agrega enlace a imagen inicio

Delgado74 committed:
Rename billetera iniciada.jpg to billetera_iniciada.jpg

Delgado74 committed:
Rename activos enviar.jpg to activos_enviar.jpg

Delgado74 committed:
Rename activos intercambiables.jpg to intercambiables.jpg

Delgado74 committed:
Rename backup frase semilla.jpg to backup_frase.jpg

Delgado74 committed:
Rename backup frase semilla2.jpg to backup_frase2.jpg

Delgado74 committed:
Rename cadenas usdt a enviar.jpg to cadenas_usdt_enviar.jpg

Delgado74 committed:
Delete src/assets/images/aqua/inicio

Delgado74 committed:
Rename recibir L-BTC.jpg to recibir_L-BTC.jpg

Delgado74 committed:
Rename recibir lightning.jpg to recibir_lightning.jpg

Delgado74 committed:
Rename recibir L-usdt.jpg to recibir_Lusdt.jpg

Delgado74 committed:
Rename recibir_L-BTC.jpg to recibir_LBTC.jpg

Delgado74 committed:
Rename recibir btc.jpg to recibir_btc.jpg

Delgado74 committed:
Rename seccion de mercado.jpg to seccion_mercado.jpg

Delgado74 committed:
Rename cadenas usdt para recibir.jpg to cadenas_usdt_recibir.jpg

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Rename billetera_iniciada.jpg to iniciada.jpg

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Add files via upload

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Rename recibir2 .jpg to recibir2.jpg

Delgado74 committed:
Delete src/assets/images/aqua/recibir1.jpg

Delgado74 committed:
Add files via upload

Delgado74 committed:
Delete src/assets/images/aqua/recibir1.jpg

Delgado74 committed:
Add files via upload

Delgado74 committed:
Delete src/assets/images/aqua/recibir1.jpg

Delgado74 committed:
Delete src/assets/images/aqua/activos_enviar.jpg

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Delete src/assets/images/aqua/ajustes.jpg

Delgado74 committed:
Delete src/assets/images/aqua/backup_frase.jpg

Delgado74 committed:
Delete src/assets/images/aqua/backup_frase2.jpg

Delgado74 committed:
Delete src/assets/images/aqua/cadenas_usdt_enviar.jpg

Delgado74 committed:
Delete src/assets/images/aqua/cadenas_usdt_recibir.jpg

Delgado74 committed:
Delete src/assets/images/aqua/enviar1.jpg

Delgado74 committed:
Delete src/assets/images/aqua/enviar2.jpg

Delgado74 committed:
Delete src/assets/images/aqua/iniciada.jpg

Delgado74 committed:
Delete src/assets/images/aqua/seccion_mercado.jpg

Delgado74 committed:
Delete src/assets/images/aqua/inicio.jpg

Delgado74 committed:
Delete src/assets/images/aqua/intercambiables.jpg

Delgado74 committed:
Delete src/assets/images/aqua/recibir depix.jpg

Delgado74 committed:
Delete src/assets/images/aqua/recibir2.jpg

Delgado74 committed:
Delete src/assets/images/aqua/recibir_LBTC.jpg

Delgado74 committed:
Delete src/assets/images/aqua/recibir_Lusdt.jpg

Delgado74 committed:
Delete src/assets/images/aqua/recibir_lightning.jpg

Delgado74 committed:
Add files via upload

Delgado74 committed:
Delete src/assets/images/aqua/recibir_btc.jpg

Delgado74 committed:
Rename activos intercambiables.jpg to intercambiables.jpg

Delgado74 committed:
Rename backup frase semilla.jpg to backup_frase.jpg

Delgado74 committed:
Rename backup frase semilla2.jpg to backup_frase2.jpg

Delgado74 committed:
Rename billetera iniciada.jpg to iniciada.jpg

Delgado74 committed:
Rename seccion de mercado.jpg to mercado.jpg

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update SUMMARY.md

Delgado74 committed:
Actualizar aqua.md

Catrya reviewed (commented):
Hola @Delgado74 gran tutorial!
Antes de poder mergearlo te recomiendo algunos cambios para que esté maś alineado con los tutoriales que ya están:
-  Pon al final la parte de ¨Autor¨, con tus redes sociales, ln address, etc
-  Pon el índice de primero, no después de la introdución como está ahora

Catrya commented on src/aqua.md:
```diff
@@ -0,0 +1,118 @@
+# Tutorial: Cómo usar la billetera AQUA
+
+AQUA es una billetera para Bitcoin y Liquid desarrollada por Jan3. Permite recibir y enviar BTC en la cadena principal, L-BTC y L-USDT, ambos tokens de la red Liquid. Además, también permite la recepción y envío de otros tokens de la red Liquid y USDT en al menos 4 cadenas más.
```
aquí sería bueno poner el link de Jan3 para que las personas sepan qué es, creo que es este https://jan3.com/ pero revisa por si acaso

Delgado74 replied:
Debería ponerlo como un vínculo en la palabra Jan3 o el link entre paréntesis? Gracias por la ayuda

Delgado74 replied:
realizado

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Update aqua.md

Delgado74 commented:
> * Pon al final la parte de ¨Autor¨, con tus redes sociales, ln address, etc
> * Pon el índice de primero, no después de la introdución como está ahora

ya estan realizadas las tareas

Delgado74 committed:
Update aqua.md

Se mejora la introducción, se agregan las secciones Conclusiones y Recomendaciones

Delgado74 committed:
Update aqua.md

Corrección final

Delgado74 committed:
Update aqua.md

Delgado74 committed:
Merge branch 'Cuba-Bitcoin:main' into aqua

Delgado74 committed:
Delete src/assets/images/aqua directory

Se elimina directorio aqua completo

Delgado74 committed:
Create 1

Delgado74 committed:
Add files via upload

Se actualiza la carpeta de imágenes para Aqua

Delgado74 committed:
Delete src/aqua.md

Se elimina aqua.md para reemplazar

Delgado74 committed:
Add files via upload

Se sube nuevo aqua.md

Delgado74 committed:
Update aqua.md

Se corrigen errores dentro del archivo MD
